list/LIST_TOPIC_README.md

```diff
@@ -0,0 +1,254 @@
+| `CompoundStatement / statusCode [@code]`                                                         | fixed value = `COMPLETE`                                                                                                                                                                                                                                                                                                         |
+| `CompoundStatement / effectiveTime`                                                              | from mapped [Encounter](../encounters/README.md) referenced in `list.encounter` <sup>1</sup><sup>2</sup><sup>3</sup>                                                                                                                                                                                                             |
+| `CompoundStatement / availabiltyTime [@value]`                                                   | `list.date` or from mapped [Encounter](../encounters/README.md) referenced in `list.encounter` <sup>4</sup><sup>5</sup>                                                                                                                                                                                                          |
+| `CompoundStatement / components`                                                                 | contains one or more `CompoundStatements` mapped from 'list.entry' <sup>6</sup>                                                                                                                                                                                                                                                  |
+
+1. When `Encounter` has `encounter.period.start` and `encounter.period.end` then values are set `effectiveTime / lowValue` & `effectiveTime / highValue` using `encounter.period.start` and `encounter.period.end` respectively 
+2. When `Encounter` has `encounter.period.start` only, then `effectiveTime / center` is set by `encounter.period.start`
+3. If there is no `encounter.period` then value `effectiveTime / center [@nullFlavour="UNK"]` is used
+4. If `list.date` is not present and `encounter.period.start` is present then that value is used 
+5. if `list.date` and `encounter.period.start` are not present `availabilityTime [@nullFlavour="UNK"]` is used
+6. each `list.entry` is mapped to `CompoundStatement` from a [List (Heading)](./LIST_HEADING_README.md) when `list.entry.reference` has a code of `24781000000107` and, when in a flat structure is mapped from the reference to the relevant clinical record
+
+<details><summary>Example XML</summary>
+
+```
+<component typeCode="COMP">
+    <CompoundStatement classCode="TOPIC" moodCode="EVN">
```

Comment:
I don't know what a `moodCode` is or whether we want to mention it in the table as a fixed value.🤔

Reply:
added